Sources/Networking/Operations/PostReceiptDataOperation.swift

```diff
@@ -53,7 +53,7 @@ final class PostReceiptDataOperation: CacheableNetworkOperation {
         /// - `subscriberAttributesByKey`
         let cacheKey =
         """
-        \(configuration.appUserID)-\(postData.isRestore)-\(postData.receiptData.hashString)
+        \(configuration.appUserID)-\(postData.isRestore)-\(postData.receipt.serialized().hashValue)
```

Comment:
Hmm this was using `receiptData.hashString` to prevent this cache key from being a huge string.
I'd recommend doing this instead now;
```
private extension EncodedAppleReceipt {

  var hash: String {
    switch self {
      case let .jws(content): return content.asData.hashString
      case let .receipt(data): return data.hashString
    }
  }

}
```

Reply:
Added as internal extension 👍